specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2019-01-01-preview/SecurityInsights.json

```diff
@@ -3774,6 +3774,115 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/{operationalInsightsResourceProvider}/workspaces/{workspaceName}/providers/Microsoft.SecurityInsights/watchlists/{watchlistAlias}/watchlistItem/{watchlistItemId}": {
```

Comment:
>watchlistItem [](start = 212, length = 13)

Resource type names in ARM are plural by convention. Also watchlistItems is kind of redundant since it's nested under a watchlists resource type. Can you rename to `watchlistItems` or just `items`? 

Also, you are missing the API to list all watchlistItems in a watchlist.

Reply:
Changed to `WatchlistItems`. 
Getting all WatchlistItems endpoint will be added in future change.

Reply:
@majastrz : could you pls review my pr ?